### ISSUE TITLE
Thunderbird Import Issue

### ISSUE THREAD

**Issue by Z3ROGRAV1TYx** [SimpleMobileTools/Simple-Contacts#440]
So if I export a vcf file from Simple-Contacts (I ensured that "Phone storage" was unchecked to avoid issues.. I tried it with it on and the names didn't even show up. Everything was completely blank).

And go into Thunderbird, Address Book and import the vcf file, the list generates of all the contacts names, but contact photos do not show up and there's no actual information in any of the contacts (All phone fields, email, etc. everything is missing. Only the names imported)

Also it added a bunch of contacts that were added as well with the name "ENCODING=b PHOTO" and they're blank as well.

**Comment by tibbi** [comment]
what app version are you using?

**Comment by Z3ROGRAV1TYx** [comment]
> 
> 
> what app version are you using?

Simple Contacts - v 6.3.5
Thunderbird (Windows 10) - v 68.1.0

**Comment by tibbi** [comment]
Try exporting a contact in a .vcf file, then opening it with any text editor. Do you see the required fields there, or the name only?

**Comment by Z3ROGRAV1TYx** [comment]
> Try exporting a contact

I'm not finding an option to export a single contact into a .vcf file. 

But if I edit the whole vcf file I exported, I see only the following fields for each contact.

```BEGIN:VCARD
VERSION:3.0
PRODID:ez-vcard 0.10.5
N:<lastname;firstname>
PHOTO;ENCODING=b;TYPE=jpeg:<HUGESTRINGOFCHARACTERS>
END:VCARD```

So no it doesn't look like it's exporting any fields besides the name (And maybe the photo? It's a huge string of characters like a few paragraphs big)

**Comment by tibbi** [comment]
you can try opening a contact and using Share, then sending your self the generated .vcf file. That basically uses the same exporting function

**Comment by Z3ROGRAV1TYx** [comment]
> 
> 
> you can try opening a contact and using Share, then sending yourself the generated .vcf file. That basically uses the same exporting function

Ah I see what you mean, yeah it does look like it worked..  kinda?

So I first imported into Signal (Messenger) and it seemed to grab the phone number, address and name, but did not grab the photo nor the birthdate I had set for the person.

Upon importing into Thunderbird doesn't seem to detect the phone number, photo or birthday. But It did grab the location (residency) and the phone number. Now the address is all on one "Address" line (StreetCity, State ZipCountry) instead of filling into their proper address, city, street, state, zip and country fields. 
_(Note it may also be missing other fields, I just don't have a lot of info for this contact I'm testing with.. So it could be losing email, notes, etc.)_

**Comment by tibbi** [comment]
you mean that the contacts are not stored under the "Phone storage (not visible by other apps)", right? They are stored casually at the shared space saved under your email, "PHONE", or so.

**Comment by tibbi** [comment]
should be fixed in 6.4.0 released soon, reopen if not

**Comment by Z3ROGRAV1TYx** [comment]
> 
> 
> you mean that the contacts are not stored under the "Phone storage (not visible by other apps)", right? They are stored casually at the shared space saved under your email, "PHONE", or so.

you mean that the contacts are not stored under the "Phone storage (not visible by other apps)", right?

When exporting I select everything EXCEPT "Phone storage (not visible by other apps)" and I have the issues listed above.

If I select "Phone storage (not visible by other apps)" upon exporting, then when importing into Thunderbird, all the imports are COMPLETELY BLANK. The names don't even show up.
